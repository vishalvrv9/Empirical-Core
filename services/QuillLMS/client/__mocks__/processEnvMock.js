export default {
  env: {
    CDN_URL: 'http://cdn.quill.org'
  }
}
