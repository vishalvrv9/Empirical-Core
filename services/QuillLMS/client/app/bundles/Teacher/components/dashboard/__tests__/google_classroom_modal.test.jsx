import React from 'react'
import { shallow } from 'enzyme'
import GoogleClassroomModal from '../google_classroom_modal.jsx'

const user={email: 'hal@spaceodyssey.org'}

import processEnvMock from '../../../../../../__mocks__/processEnvMock.js';
window.process = processEnvMock;

describe('the GoogleClassroomModal component', () => {

  it('should render', () => {
      const wrapper = shallow(
        <GoogleClassroomModal user={user} />
      )
      expect(wrapper).toMatchSnapshot()
  })

})
