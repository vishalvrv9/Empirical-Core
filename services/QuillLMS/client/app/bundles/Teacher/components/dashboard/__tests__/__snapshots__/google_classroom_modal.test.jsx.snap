// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the GoogleClassroomModal component should render 1`] = `
<Modal
  animation={true}
  autoFocus={true}
  backdrop={true}
  bsClass="modal"
  dialogClassName="google-classroom-modal"
  dialogComponentClass={[Function]}
  enforceFocus={true}
  keyboard={true}
  manager={
    ModalManager {
      "containers": Array [],
      "data": Array [],
      "handleContainerOverflow": true,
      "hideSiblingNodes": true,
      "modals": Array [],
    }
  }
  onHide={[Function]}
  renderBackdrop={[Function]}
  restoreFocus={true}
  show={false}
  user={
    Object {
      "email": "hal@spaceodyssey.org",
    }
  }
>
  <ModalBody
    bsClass="modal-body"
    componentClass="div"
  >
    <img
      alt="close-modal"
      className="pull-right react-bootstrap-close"
      src="http://cdn.quill.org/images/shared/close_x.svg"
    />
    <h1
      className="q-h2"
    >
      First: Update Your Email
    </h1>
    <p>
      Your Quill email address must be the same as your Google Classroom email. If your email is the same click Next. If it is different, you can update your Quill email below.
    </p>
    <_class
      email="hal@spaceodyssey.org"
    />
    <a
      className="q-button cta-button bg-quillgreen text-white"
      href="/auth/google_oauth2"
    >
      Next: Choose Google Classrooms
    </a>
  </ModalBody>
</Modal>
`;
